docs/api/cookies.md

```diff
@@ -13,21 +13,30 @@ For example:
 const { session } = require('electron')
 
 // Query all cookies.
-session.defaultSession.cookies.get({}, (error, cookies) => {
-  console.log(error, cookies)
-})
+session.defaultSession.cookies.get({})
+  .then((cookies) => {
```

Comment:
we may want to use async await for these since we don't do anything with the cookies in the microtask

Reply:
I restored the original `console.log()` from the original example. e27cbd7d1bbe3d044a9fa55a6b19f7e482e736df